src/powershell/Tests/Unit/ConvertTo-FinOpsSchema.Tests.ps1

```diff
@@ -0,0 +1,98 @@
+# Add Pester tests
+$PesterPreference = [PesterConfiguration]::Default
+$PesterPreference.Output.Verbosity = 'Detailed'
+# . source the script
+. ..\Public\ConvertTo-FinOpsSchema\ConvertTo-FinOpsSchema.ps1
+
+
+
+# Check the current working directory
+# Get-Location
+
+# Set the working directory if needed
+# Set-Location ".\"
+
+
+#### Use Execution policy only if needed (e.g. if the script is not signed, etc.) ###
+# Check the execution policy
+# Get-ExecutionPolicy
+# Set the execution policy to RemoteSigned if needed
+# Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Force
+
+
+# Install/Update Pester if needed
+# Check Pester version
+Get-Module Pester -ListAvailable
+# Install/Update Pester
+Install-Module Pester -Force -SkipPublisherCheck -Scope CurrentUser
+
+Describe "ConvertTo-FinOpsSchema" {
+    BeforeAll {
+        . $PSCommandPath.Replace('.ConvertTo-FinOpsSchema.ps1', '.ps1')
+        # Set the script scope variables
+        $script:ActualCost = @(
```

Comment:
FYI - If you want a file to use for testing, we can add a few CSVs. Any of the Trey Research data is good. I was already thinking about adding some of that as sample data to the repo anyway.

Reply:
I have csv's. I will add them. Can I add it in the same directory?

Reply:
That's a good starting point. I was debating on adding it as sample data that is just part of the repo for people to reference as a learning resource.